bin/zktls-pairs/src/main.rs

```diff
@@ -1,17 +1,37 @@
 extern crate mock_lib;
 
 use automata_sgx_sdk::types::SgxStatus;
+use clap::{Arg, Command};
 
 automata_sgx_sdk::enclave! {
     name: Enclave,
     ecall: {
-        fn trusted_execution() -> SgxStatus;
+        fn trusted_execution(file_path_ptr: *const u8, file_path_len: u32) -> SgxStatus;
```

Comment:
you changed `file_path_len` from `usize` to `u32` because you are casting to this type?
okay, we can keep it like this, but I meant just to delete cast `as u32` from row 34